mayavi/core/pipeline_base.py

```diff
@@ -121,10 +121,10 @@ def render(self):
                 if hasattr(actor, 'mapper'):
                     m = actor.mapper
                     if m is not None:
-                        m.update(0)
+                        m.update()
```

Comment:
I am not sure, I think this was when there was issues with the old and new pipeline. The current test failures have nothing to do with this. Those have to do with the import of a UI toolkit when tvtk is imported -- this could be in some other library.